solutions.txt

```diff
@@ -64,23 +62,23 @@ Exercise 6:
 
 // Complete Exercise 6 below...
 
-foods [1] = ('sushi');
-foods [2]= ('cupcake');
-foods.push ('tofu');
-
+foods.pop();
+foods.pop();
+foods.pop();
+foods.push('sushi', 'cupcake','tofu', 'cheeseburger');
+console.log('Exercise 6 Result:\n', foods)
+//I tried using the .replace function but didnt seem to get the result I wanted. Again, did a lot of googling but felt like I still was not understanding fully. This probably doesnt align with the lecture but I tired to match it as close as possible.
```

Comment:
again, this works great! If you want to talk through the `replace` method, come by my office hours so we can talk through it